src/ServerRequestFactoryInterface.php

```diff
@@ -10,10 +10,20 @@
     /**
      * Create a new server request.
      *
-     * @param string $method
-     * @param UriInterface|string $uri
+     * The method and URI of the request SHOULD be derived from the given
+     * server parameters.
+     *
+     * If the method or URI parameter is passed, these values MUST be used
+     * instead of the server parameters.
```

Comment:
@weierophinney I feel like this docblock could be worded better. Any suggestions?